index.html

```diff
@@ -61,5 +61,203 @@ <h3>I’am a Software Developer</h3>
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <div class="btn-group flex">
+                <button class="btn">css</button>
+                <button class="btn">html</button>
+                <button class="btn">bootstrap</button>
+                <button class="btn">ruby</button>
+              </div>
+            </div>
+            <button class="special">See Project</button>
+          </div>
+          <div class="card-details">
+            <img src="asset/card.png" alt="image" />
+            <div class="card-content on-hover">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <div class="btn-group flex">
+                <button class="btn">css</button>
+                <button class="btn">html</button>
+                <button class="btn">bootstrap</button>
+                <button class="btn">ruby</button>
+              </div>
+            </div>
+            <button class="special">See Project</button>
+          </div>
+          <div class="card-details">
+            <img src="asset/card.png" alt="image" />
+            <div class="card-content on-hover">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <div class="btn-group flex">
+                <button class="btn">css</button>
+                <button class="btn">html</button>
+                <button class="btn">bootstrap</button>
+                <button class="btn">ruby</button>
+              </div>
+            </div>
+            <button class="special">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-content flex">
+          <h2>About me</h2>
+          <div>
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <button class="special">Get My Resume</button>
+          </div>
+        </div>
+        <img src="asset/bottomillus.png" alt="image" class="illustration" />

```

Comment:
- [ ] Instead of adding this illustration/graphic as an image, consider adding it with CSS as a background image (decorations are always backgrounded images, not `img` tags.)

Reply:
As an image or background, does it really matter?

I believe it will add more syntax on the code, pls take a look at it again from this perspective